wgpu-hal/src/vulkan/device.rs

```diff
@@ -719,6 +719,14 @@ impl super::Device {
         self.shared.physical_device
     }
 
+    pub fn raw_queue(&self) -> ash::vk::Queue {
+        unsafe {
+            self.shared
+                .raw
+                .get_device_queue(self.queue_family_index(), self.shared.queue_index)
```

Comment:
Since we already have the queue, can we avoid the extra call here in some way?

Reply:
I couldn't find a way, but that doesn't mean that there isn't any. The code isn't exactly easy to read and there are no comments and no developer documentation.

Reply:
If you look at where DeviceShared is initialized, you'll see a raw_queue variable.

Although I still think the index should be exposed as well

Reply:
I added this to the pull request, hope that this is ok. Since `ash::vk::Queue` is `Copy`, this shouldn't be an issue.

Reply:
Yep that's fine.